packages/remark-parse/lib/tokenizer.js

```diff
@@ -6,27 +6,23 @@
  * @fileoverview Markdown tokenizer.
  */
 
-'use strict';
+export default factory;
```

Comment:
Remember that ESM exports are staticly defined. You shouldn't export before the object/function has been allocated or it will export `undefined` in strict loaders.